pkg/service/HelmAppService.go

```diff
@@ -661,8 +662,12 @@ func (impl HelmAppServiceImpl) TemplateChart(ctx context.Context, request *clien
 		MaxHistory:    0,    // limit the maximum number of revisions saved per release. Use 0 for no limit (default 10)
 		RepoURL:       request.ChartRepository.Url,
 	}
-	HelmTemplateOptions := &helmClient.HelmTemplateOptions{}
 
+	HelmTemplateOptions := &helmClient.HelmTemplateOptions{
+		KubeVersion: &chartutil.KubeVersion{
+			Version: request.K8SVersion,
```

Comment:
can we make this conditional, if version not provided don't add this

Reply:
done